unity-client/Assets/Scripts/MainScripts/DCL/ServiceProviders/Catalyst/Catalyst.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using DCL;
+using DCL.Helpers;
+using Variables.RealmsInfo;
+
+public interface ICatalyst : IDisposable
+{
+    Promise<SceneDeploymentPayload> GetDeployedScenes(string[] parcels, bool onlyCurrentlyPointed = true, string sortBy = null, string sortOrder = null);
+    Promise<string> GetDeployments(DeploymentOptions deploymentOptions);
+    Promise<string> Get(string url);
+}
+
+public class Catalyst : ICatalyst
+{
+    private const int CACHE_TIME_MSECS = 5 * 60 * 1000;
```

Comment:
miliseconds for cache can be an overkill, are you thinking in a special case where you will need the milliseconds there? instead of seconds/minutes?

Reply:
You remind me I have to check something related to that :)
It's using milliseconds cause the task delay it's expecting milliseconds. But the caching here is set for 5 minutes